dotcom-rendering/src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -392,6 +402,21 @@ export const ReaderRevenueLinks: React.FC<Props> = ({
 	ophanRecord,
 	pageViewId = '',
 }: Props) => {
+	const [countryCode, setCountryCode] = useState<string>();
+
+	useEffect(() => {
```

Comment:
I want to suggest a custom hook for this, but with only 2 uses it's maybe not worth it!

Reply:
I have exactly the same instinct. Perhaps the real benefit to having a custom hook is in terms of dev ex? It might be nice to have a suite of hooks or functions that provide components with the state that they need to achieve their aims.

I'm going to draw up a list of these for review.